src/NavBar.js

```diff
@@ -0,0 +1,32 @@
+import {
+  BrowserRouter as Router, Link, Switch, Route,
+} from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
+
+const NavBar = () => (
+  <Router>
+    <div>
+      <nav className="navbar navbar-light bg-light">
+        <ul className="navbar-nav">
+          <li className="nav-item">
+            <Link to="/" class="nav-link">Books</Link>
+          </li>
+          <li className="nav-item">
+            <Link to="/categories" class="nav-link">Categories</Link>
+          </li>
+        </ul>
+      </nav>
+      <Switch>
+        <Route exact path="/">
```

Comment:
- [ ] The `Router` and `Switch` and `Route` should all be applied within the App.js file and not the navbar component. Make this necessary change.